scene/gui/line_edit.h

```diff
@@ -259,26 +259,26 @@ class LineEdit : public Control {
 	void set_placeholder_alpha(float p_alpha);
 	float get_placeholder_alpha() const;
 
-	void set_cursor_position(int p_pos);
-	int get_cursor_position() const;
+	void set_caret_position(int p_pos);
+	int get_caret_position() const;
 
 	void set_max_length(int p_max_length);
 	int get_max_length() const;
 
-	void append_at_cursor(String p_text);
+	void append_at_caret(String p_text);
 	void clear();
 
 	void set_mid_grapheme_caret_enabled(const bool p_enabled);
 	bool get_mid_grapheme_caret_enabled() const;
 
-	bool cursor_get_blink_enabled() const;
-	void cursor_set_blink_enabled(const bool p_enabled);
+	bool is_caret_blink_enabled() const;
+	void set_caret_blink_enabled(const bool p_enabled);
 
-	float cursor_get_blink_speed() const;
-	void cursor_set_blink_speed(const float p_speed);
+	float get_caret_blink_speed() const;
+	void set_caret_blink_speed(const float p_speed);
 
-	bool cursor_get_force_displayed() const;
-	void cursor_set_force_displayed(const bool p_enabled);
+	bool get_caret_force_displayed() const;
```

Comment:
As it's a bool `is_caret_force_displayed` sounds better to me.

Reply:
Done.
I've also similarly updated the getters and setters for `expand_to_text_length` and `caret_mid_grapheme`.